src/System.Linq.Expressions/src/System/Dynamic/DynamicObject.cs

```diff
@@ -764,8 +764,8 @@ private bool IsOverridden(string method)
 
                             if (baseMethod != null)
                             {
-                                ParameterInfo[] baseParams = baseMethod.GetParameters();
-                                ParameterInfo[] miParams = mi.GetParameters();
+                                ParameterInfo[] baseParams = baseMethod.GetParametersCached();
+                                ParameterInfo[] miParams = mi.GetParametersCached();
```

Comment:
Reverting this change to avoid conflicts with https://github.com/dotnet/corefx/pull/14116.